packages/expo-updates/src/Updates.ts

```diff
@@ -73,7 +73,13 @@ export const localAssets: LocalAssets = ExpoUpdates.localAssets ?? {};
  * otherwise. If you are concerned about backwards compatibility of future updates to your app, you
  * can use this constant to provide special behavior for this rare case.
  */
-export const isEmergencyLaunch: boolean = ExpoUpdates.isEmergencyLaunch || false;
+export const isEmergencyLaunch = ExpoUpdates.isEmergencyLaunch;
+
+/**
+ * If `isEmergencyLaunch` is set to true, this will contain a string error message describing
+ * what failed during initialization.
+ */
+export const emergencyLaunchReason = ExpoUpdates.emergencyLaunchReason;
```

Comment:
Could we also add this to the currentlyRunning object returned by the useUpdates hook?